packages/webpack-cli/__tests__/info.test.js

```diff
@@ -1,3 +1,4 @@
+//eslint-disable-next-line node/no-extraneous-require
```

Comment:
I think we need to improve config for this rule

Reply:
Hmm, we have disabled this rule at many places.

Reply:
Yep, it is not good, let's improve this

Reply:
I think we can `disable` it for test files.

Reply:
@snitin315 Ideally we have all tests packages in devDeps, so it should be not triggered, need investigate

Reply:
Maybe we need to use [`allowModules`](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-require.md#allowmodules) ->

```js
'node/no-extraneous-require': ['error', {
            'allowModules': ['execa', 'webpack', 'webpack-dev-server']
        }]
```

Reply:
Why do warning this rule? We have this in our dev deps

Reply:
No idea why it's triggering 😢

Reply:
Can you look at rule logic?

Reply:
This one is not specified explicitly in rules.
https://github.com/webpack/webpack-cli/blob/2c390a1fdcbb85b7596c7b621c8d263da1734690/.eslintrc.js#L11-L18

Reply:
I mean inside code of rule and check why we got error here

Reply:
https://github.com/mysticatea/eslint-plugin-node/blob/4a348e7d24e0d806d7873f293c91a922d0316d14/lib/util/check-extraneous.js#L27-L34

I logged the dependencies set here and found its collecting deps defined inside `packages/webpack-cli/package.json` and not root `package.json` -

![Screenshot at 2020-10-09 15-53-40](https://user-images.githubusercontent.com/46647141/95572487-a6a94900-0a47-11eb-8a35-c55afe76d2e3.png)

Reply:
I think we should create an issue for the same in the `eslint-plugin-node` repo.

Reply:
Oh, I think we need setup `resolvePaths` (https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-extraneous-require.md, for `import` too) and setup root `node_modules` here

Reply:
```
 "settings": {
        "node": {
            "resolvePaths": [__dirname],
        }
    },
```

let try in `eslint` root config

Reply:
still the same :confused:

Reply:
Can you check different values and look at source code of the rule

Reply:
In near future 👍